system-test/mdbci/run_test.sh

```diff
@@ -81,7 +81,9 @@ else
     eval "arguments=(${test_set})"
     ctest -N "${arguments[@]}"
     ctest -VV "${arguments[@]}"
-    ctest --rerun-failed -VV
+    if [ -f "Testing/Temporary/LastTestsFailed.log" ]; then
+        ctest --rerun-failed -VV
```

Comment:
This line can simply be removed, rerunning the failed tests has caused more problems than it has solved.